cmd/bundle/generate/utils.go

```diff
@@ -101,10 +108,8 @@ func (n *downloader) markNotebookForDownload(ctx context.Context, notebookPath *
 		return err
 	}
 
-	ext := notebook.GetExtensionByLanguage(info)
```

Comment:
Did you mean to stop adding the extension here?

Reply:
it is still added just in the same line below